crocks/List.spec.js

```diff
@@ -119,6 +120,14 @@ test('List value', t => {
   t.end()
 })
 
+test('List toArray', t => {
+  const a = List([ 'some-thing', ['else', 43], 34 ]).toArray()
+
+  t.same(a, [ 'some-thing', 'else', 43, 34 ], 'folds the wrapped array')
```

Comment:
This is a BAD test and actually shows this is broken. (the issue is your reduce and how you are doing concat), but that will get fixed with `.slice()`

```js
const data = [ 'some-thing', ['else', 43], 34 ]
const a = List(data)

t.same(a, data, 'folds the wrapped array')
```

The inner structure MUST match exactly.